<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsPanel</class>
 <widget class="QDockWidget" name="ToolsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>90</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>90</width>
    <height>524287</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="skelDeformation">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Skel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cageDeformation">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cage</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCamera">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreCamera">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cageUpdater">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CgUpdater</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skelUpdater">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SkUpdater</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>70</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exportCamera">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importCamera">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>280</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
